addons/mrp/migrations/12.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_renames = {
+    'mrp_workcenter_productivity_loss': [
+        ('loss_type', None),
```

Comment:
You mustn't renamed this column, as the new field is related stored.

Reply:
Oh, yes, I saw wrong when I did it. I thought it wasn't stored.

Reply:
maybe `copy_columns`? I think I wanted to save the column because I feared it would change the value in the loading of data because it is a related.

Reply:
No if the column exists. It happens the same as stored computed fields.